public_html/wp-content/plugins/wc-post-types/wc-post-types.php

```diff
@@ -1495,13 +1535,16 @@ public function save_post_session( $post_id, $post ) {
 				sanitize_text_field( $_POST['wcpt-session-date'] ),
 				absint( $_POST['wcpt-session-hour'] ),
 				absint( $_POST['wcpt-session-minutes'] ),
-				'am' == $_POST['wcpt-session-meridiem'] ? 'am' : 'pm'
+				'am' === $_POST['wcpt-session-meridiem'] ? 'am' : 'pm'
 			) );
 			update_post_meta( $post_id, '_wcpt_session_time', $session_time );
 
+			update_post_meta( $post_id, '_wcpt_session_length_hours',   absint( $_POST['wcpt-session-length-hours']   ) );
+			update_post_meta( $post_id, '_wcpt_session_length_minutes', absint( $_POST['wcpt-session-length-minutes'] ) );
```

Comment:
Same comment as on the API side - I think it would make more sense to calculate the duration (in seconds, probably) and save that in one single field.

Reply:
Done in 4aa7acb286d75469cc05253ee976fb5f38ed05f4